docs/generate_example_images.py

```diff
@@ -12,7 +13,7 @@
 import kornia as K
 
 
-def read_img_from_url(url: str, resize_to: Optional[Tuple[int, int]] = None) -> torch.Tensor:
+def read_img_from_url(url: str, resize_to: tuple[int, int] | None = None) -> torch.Tensor:
```

Comment:
This king of change does not look right...

Reply:
Why? This type annotation is from 3.9 or 3.10 and we can use it since we add `from __future__ import annotations`  -- [pep604](https://peps.python.org/pep-0604/)

Reply:
Mostly because our main reason to have annotations is to make kornia code jittable. I am not an expert in type annotations, but as long as `torch.jit` itself uses "old-style" annotations, we should use them as well
https://github.com/pytorch/pytorch/blob/master/torch/_jit_internal.py#L21